### PR TITLE
Adds possibility to define get matching xpath count kw return type

### LINKED ISSUE #715: Support returning int from `Get Matching Xpath Count`
This "feature" has been present AFAIK since about the first check-in. I tripped up over this when attempting to add a value to the return and got a type error.

Two suggested approaches:

1. Change the documentation, per [this pull request](https://github.com/robotframework/Selenium2Library/pull/714).
2. As an optional keyword argument, allow for an `int` return (default would remain `str`). I have not included a code example/pull for that.

### PR DESCRIPTION
**Pull request by aaltat**
Fixes #715

### CODE REVIEW

File: src/Selenium2Library/keywords/_element.py [hunk 1]
```diff
@@ -622,9 +622,12 @@ def page_should_not_contain_image(self, locator, message='', loglevel='INFO'):
 
     # Public, xpath
 
-    def get_matching_xpath_count(self, xpath):
+    def get_matching_xpath_count(self, xpath, return_str=True):
         """Returns number of elements matching `xpath`
 
+        The default return type is `str`but it can changed to `int` by setting
```
Comment (kormbrek-rally) [inline src/Selenium2Library/keywords/_element.py]: missing space after \`str\`

File: src/Selenium2Library/keywords/_element.py [hunk 2]
```diff
@@ -636,7 +639,7 @@ def get_matching_xpath_count(self, xpath):
         `Xpath Should Match X Times`.
         """
         count = len(self._element_find("xpath=" + xpath, False, False))
-        return str(count)
+        return str(count) if return_str else count
```
Comment (kormbrek-rally) [inline src/Selenium2Library/keywords/_element.py:642]: IMHO we should follow [DateTime's](https://github.com/robotframework/robotframework/blob/master/src/robot/libraries/DateTime.py) lead and use a util method to make sure that `str('false')` is treated as `False`, etc.
```
from robot.utils import is_truthy
...
return str(count) if is_truthy(return_str) else count
```

File: test/acceptance/keywords/elements.robot [hunk 3]
```diff
@@ -61,6 +62,12 @@ Get Matching XPath Count
     [Documentation]    Get Matching XPath Count
     ${count}=    Get Matching XPath Count    //a
     Should Be Equal    ${count}    19
+    ${count}=    Get Matching XPath Count    //a    ${True}
+    Should Be Equal    ${count}    19
+    Should Be String    ${count}
+    ${count}=    Get Matching XPath Count    //a    ${False}
+    Should Be Equal As Numbers    ${count}    19
```
Comment (kormbrek-rally) [inline test/acceptance/keywords/elements.robot]: I think you can just do a direct comparison to an `int` here...
```
Should Be Equal    ${count}    ${19}
```

File: src/Selenium2Library/keywords/_element.py (hunk 2 above)
Comment (aaltat) [reply src/Selenium2Library/keywords/_element.py:642]: I agree, but would like to do it same time for all the keywords using booleans. There is #719 for this problem.

File: test/acceptance/keywords/elements.robot (hunk 3 above)
Comment (aaltat) [reply test/acceptance/keywords/elements.robot]: Good point, I will fix it.

File: src/Selenium2Library/keywords/_element.py (hunk 1 above)
Comment (aaltat) [reply src/Selenium2Library/keywords/_element.py]: Good catch, I will fix it.

**Pull request merged**